### PR TITLE
fix: displaying expiry date when issuing a vc

### PR DESCRIPTION
**Pull request by dawidgil**
https://energyweb.atlassian.net/browse/SWTCH-1566

### CODE REVIEW

**Review by whitneypurdum — APPROVED** [review]

File: src/app/shared/expiration/date-picker/expiration-date.component.ts [hunk 1]
```diff
@@ -19,14 +19,24 @@ export class ExpirationDateComponent implements OnInit, OnDestroy {
   /**
    * Defined in milliseconds.
    */
-  @Input() defaultValidityPeriod: number;
+  @Input() set defaultValidityPeriod(period: number) {
+    if (!period) {
+      return;
+    }
+    this._defaultPeriod = period;
+    this.defaultExpirationDate();
```
Comment (jrhender) [inline src/app/shared/expiration/date-picker/expiration-date.component.ts:27]: Maybe this is a side point, but I find understanding the difference between `defaultExpirationDate` and `setDefaultExpirationDate` challenging. It seems like `setDefaultExpirationDate` has the same code as `defaultExpirationDate`, maybe `setDefaultExpirationDate` could call `defaultExpirationDate` and the function names could be adjusted accordingly?

Comment (dawidgil) [reply src/app/shared/expiration/date-picker/expiration-date.component.ts:27]: ahh, yeah. Code looks similar. I will adjust it, thanks for pointing this out.

Comment (dawidgil) [reply src/app/shared/expiration/date-picker/expiration-date.component.ts:27]: https://github.com/energywebfoundation/switchboard-dapp/pull/783 hope this is what you wanted :)

Comment (jrhender) [reply src/app/shared/expiration/date-picker/expiration-date.component.ts:27]: Thanks!

**Pull request merged**